llvm/test/Transforms/MergeFunc/mergefunc-preserve-debug-info.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
```

Comment:
Please pre-commit the test check regeneration, so that this shows a meaningful diff.